kubernetes/docs/includes/deploy-jars-k8s.rst

```diff
@@ -1,36 +1,23 @@
-Deploying Custom Kafka Connect JARs with Operator 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Deploying Connectors with Operator 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-`Kafka Connect <https://docs.confluent.io/current/connect/index.html>`__ employs an extensible programming model to faciliate `Data Conversions <https://docs.confluent.io/current/connect/concepts.html#converters>`__, `Single Message Transformations (SMTs) <https://docs.confluent.io/current/connect/concepts.html#transforms>`__, or the development of custom connectors if a `supported connector <https://www.confluent.io/hub/>`__ is not available.  The Connect `packaging model <https://docs.confluent.io/current/connect/devguide.html#packaging>`__ will require that you deploy an archive containing your custom JAR to the `Kafka Connect's plugin path <https://docs.confluent.io/current/connect/userguide.html#connect-installing-plugins>`__.
+|c-hub| is an online library of pre-packaged and ready-to-install connectors, transformations, and converters that suit your needs.
+The |co-long| image for |kconnect-long|, `confluentinc/cp-server-connect-operator <https://hub.docker.com/r/confluentinc/cp-server-connect-operator>`__,  includes a small number of those connectors but may not have the specific connector you want to deploy.
+Therefore, to deploy a new connector in your Kubernetes environment, you will need to create a custom Docker image that bundles the desired connector jars into the Connect image.
+See the `documentation <https://docs.confluent.io/current/connect/managing/extending.html#create-a-docker-image-containing-c-hub-connectors>`__ to learn how to use the |c-hub| client to create a custom Docker image that extends one of Confluent’s Kafka Connect images with a specific set of ready-to-install connectors.
+Once you build the custom Docker image, Kubernetes will need to pull this image from a Docker Registry to create the Pods.
 
-This demonstration utilizes a `custom Connector to generate mock events <https://github.com/confluentinc/kafka-connect-datagen>`__ which is **not** installed by default in the Confluent Kafka Connect Docker images.  In order to utilize this plugin, we build a custom Docker image which is derived from the Confluent Operator based Docker image and install the custom connector using the ``confluent-hub`` client.
+For more advanced use cases where you want to use a custom connector instead of a pre-packaged one available at |c-hub|, you may create a Docker image with a custom connector from a `local archive <https://docs.confluent.io/current/connect/managing/confluent-hub/command-reference/confluent-hub-install.html#confluent-hub-client-install>`__.
```

Comment:
I like this but I also liked the text that discusses Converters and SMTs, as their JARs also need to be deployed even if it's not a full custom connector.   As it's the same exact problem/workflow, can we add back in some language about those features of Connect as well to try and help any readers that end up here looking to deploy a custom SMT w/ Operator?

Reply:
Please see latest commit, I added the links back in (at the top)